deployment/fetch_vnodes_log.sh

```diff
@@ -75,6 +83,12 @@ parseArguments(){
             -b | --ON_BUILD_CONFIG_DIR )    shift
                                             ON_BUILD_CONFIG_DIR=$1
                                             ;;
+            -n | --NOHUP )                  shift
+                                            NOHUP=$1
+                                            ;;
+            -b | --BUILD_ID )               shift
```

Comment:
missing this two parameter in Usage()
and you can make NOHUP an optional parameter (default is false)

Reply:
It's an optional parameter now， default default value is "" for shell.